products/nomad.md

```diff
@@ -15,15 +15,15 @@ releases:
   - releaseCycle: "1.2"
     eol: false
     release: 2021-11-15
-    latest: "1.2.0"
+    latest: "1.2.1"
   - releaseCycle: "1.1"
     eol: false
     release: 2021-05-18
-    latest: "1.1.7"
+    latest: "1.1.8"
   - releaseCycle: "1.0"
     eol: false
     release: 2020-12-08
-    latest: "1.0.13"
+    latest: "1.0.14"
   - releaseCycle: "0.12"
     eol: false
```

Comment:
If i am not wrong branch reached eol so could you mark this one as `eol: true`

Reply:
The documentation states that GA releases are supported for up to 2 years, therefore I didn't mark it EOL yet. However, since the 0.X-branch was never declared GA, I believe you are right that this cycle can be marked EOL. I have made the requested changes.